frontend/tablet/src/components/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { StyleSheet, Text } from 'react-native'
+import { TouchableOpacity } from 'react-native-gesture-handler'
+
+interface Props {
+  title: string
+  onPress: () => void
+}
+
+const Button: React.FC<Props> = (props) => {
+  return (
+    <TouchableOpacity style={styles.container} onPress={props.onPress}>
+      <Text style={styles.text}>{props.title}</Text>
```

Comment:
You will need a `Typography` component and move the `styles.text` there. That way we will have all of the different fonts and their style variations in 1 single file.